homeassistant/components/avri/sensor.py

```diff
@@ -72,13 +55,7 @@ def name(self):
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
-        return (
-            f"{self._waste_type}"
-            f"-{self._client.country_code}"
-            f"-{self._client.postal_code}"
-            f"-{self._client.house_nr}"
-            f"-{self._client.house_nr_extension}"
-        )
+        return (f"{self._integration_id}" f"-{self._waste_type}").replace(" ", "")
```

Comment:
If the unique id is changing this should be listed in the breaking changes section with instructions on what to do with the orphaned entities.

Reply:
I added somes notes in the PR for this.

Reply:
We should tell the user how to delete the old orphaned entities.

Reply:
I added a bit more info about removing orphaned entities. It's the same process basically as any other orphaned entity.